sympy/matrices/tests/test_eigen.py

```diff
@@ -681,3 +681,9 @@ def test_issue_20275():
         [Matrix([[S(1)/2 + sqrt(5)/2], [0], [1], [1], [0]]),
          Matrix([[S(1)/2 + sqrt(5)/2], [1], [0], [0], [1]])]
     )
+
+
+def test_issue_20752():
+    b = symbols('b',nonzero=True)
+    m = Matrix([[0, 0, 0], [0, b, 0], [0, 0, b]])
+    assert m.is_positive_semidefinite == None
```

Comment:
There should be a space after a comma and also `== None` is better as `is None`.

Reply:
I've made changes but can this test go with similar others : ?
https://github.com/sympy/sympy/blob/cf846222b245629f331c9ad38b80843f90cc0d56/sympy/matrices/tests/test_eigen.py#L505-L599